config/jobs/kubernetes-sigs/sig-windows/release-master-windows.yaml

```diff
@@ -178,6 +189,11 @@ periodics:
     base_ref: master
     path_alias: sigs.k8s.io/windows-testing
     workdir: true
+  - org: kubernetes-sigs
```

Comment:
@marosset @jsturtevant fyi this won't affect anything for all the sig windows jobs use templates in https://github.com/kubernetes-sigs/windows-testing/tree/master/capz/templates until we switch those to external cloud-provider, but then we'll need to also update az capi or the windows-testing script to build the CCM images just like ci-conformance.sh does.

Reply:
Will also need to do https://github.com/kubernetes/test-infra/pull/28825 for those to have a registry we can push and pull the CCM images from